### PR TITLE
add id_liputan6 dataset

### PR DESCRIPTION
**Pull request by cahya-wirawan**
id_liputan6 is a large-scale Indonesian summarization dataset. The articles were harvested from an online news portal, and obtain 215,827 document-summary pairs: https://arxiv.org/abs/2011.00679

### CODE REVIEW

File: datasets/id_liputan6/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+an online news portal, and obtain 215,827 document-summary pairs. We leverage pre-trained language models to develop
+benchmark extractive and abstractive summarization methods over the dataset with multilingual and monolingual
+BERT-based models. We include a thorough error analysis by examining machine-generated summaries that have
+low ROUGE scores, and expose both issues with ROUGE it-self, as well as with extractive and abstractive
+summarization models.
+
+You need to manually request the liputan6 dataset using the form in https://github.com/fajri91/sum_liputan6/
+and uncompress it. The liputan6 dataset can then be loaded using the following command 
+`datasets.load_dataset("id_liputan6", 'canonical', data_dir="<path/to/uncompressed_folder>")` or
+`datasets.load_dataset("id_liputan6", 'xtreme', data_dir="<path/to/uncompressed_folder>")`.
```
Comment (lhoestq) [inline datasets/id_liputan6/README.md:69]: Could you explain the difference between the two ?

**Review by lhoestq — COMMENTED** [review]
Really cool thank you !
Thanks for cleaning/detokenizing the text

I left a few comments

File: datasets/id_liputan6/README.md (hunk 1 above)
Comment (cahya-wirawan) [reply datasets/id_liputan6/README.md:69]: According to the paper, the author  discarded in xtreme variant the development and test document–summary pairs where the summary has fewer than 90% novel 4-grams (page 3 in https://www.aclweb.org/anthology/2020.aacl-main.60.pdf).
I put also this info in readme file.

File: datasets/id_liputan6/id_liputan6.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        data_dir = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
```
[the remark this answers is not in the document]
Comment (cahya-wirawan) [reply datasets/id_liputan6/id_liputan6.py:108]: I added it the new update

File: datasets/id_liputan6/README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  'id': 'string',
+  'url': 'string',
+  'clean_article': 'string',
+  'clean_article': 'string',
+  'extractive_summary': 'string'
+}
+```
+### Data Instances
+
+[More Information Needed]
```
[the remark this answers is not in the document]
Comment (cahya-wirawan) [reply datasets/id_liputan6/README.md:89]: done in the new update. Thanks

**Pull request merged**